### DISCUSSION TITLE
Does Apollo Server Support Fragment Variables?

### DISCUSSION THREAD

**Discussion by howlettga** [apollographql/apollo-server#7996] [Q&A]
It seems like in older versions, Apollo Server supported Fragment Variables via `parseOptions`:

```
{
  parseOptions: {
    experimentalFragmentVariables: true,
  }
}

These are simply not supported now right? I see that the graphql client does not directly support this option and has to be accessed via `graphql-tag`.

**Comment by howlettga** [comment]
More information here: https://github.com/apollographql/graphql-tag#experimental-fragment-variables
and here: https://github.com/graphql/graphql-wg/discussions/1239

**Comment by glasser** [comment]
I don't believe Apollo Server ever had any specific support for this flag or feature. `parseOptions` is passed through to the underlying `graphql` (aka `graphql-js`) library, so perhaps some older version or fork of that library once supported `experimentalFragmentVariables`?
